### PR TITLE
Update all pipelines to use partial responses

### PR DESCRIPTION
**Pull request by deflaux**
Also:
* Fixed a bug in CalculateCoverage and filed https://github.com/googlegenomics/dataflow-java/issues/186
* Updated AnnotateVariants pipeline to use gRPC and fixed https://github.com/googlegenomics/dataflow-java/issues/79

### CODE REVIEW

**Comment by dionloy** [comment]
Reviewing now. Very interested in kicking the tires once this is in to see how much of a difference it makes in walltime!

File: src/main/java/com/google/cloud/genomics/dataflow/pipelines/AnnotateVariants.java [hunk 1]
```diff
@@ -141,21 +143,17 @@ public AnnotateVariants(OfflineAuth auth,
...
+      DoFn<StreamVariantsRequest, KV<String, VariantAnnotation>>.ProcessContext c) throws Exception {
     Genomics genomics = GenomicsFactory.builder().build().fromOfflineAuth(auth);
 
-    SearchVariantsRequest request = c.element();
+    StreamVariantsRequest request = StreamVariantsRequest.newBuilder(c.element())
+        .addAllCallSetIds(callSetIds)
+        .build();
     LOG.info("processing contig " + request);
-    Iterable<Variant> varIter = FluentIterable
-        .from(Paginator.Variants.create(genomics, ShardBoundary.Requirement.STRICT).search(
```
Comment (dionloy) [inline src/main/java/com/google/cloud/genomics/dataflow/pipelines/AnnotateVariants.java]: should we deprecate Paginator ?  or update docs to strongly encourage the Stream* case instead? (doesn't have to be in this PR).

Comment (deflaux) [inline src/main/java/com/google/cloud/genomics/dataflow/pipelines/AnnotateVariants.java]: Yes.  There are two relevant commits in this PR (1) add the integration test to check existing functionality (2) update to streaming confirming that integration test still passes.

**Comment by dionloy** [comment]
LGTM

Thanks!

**Pull request merged**